sycl/source/detail/scheduler/scheduler.hpp

```diff
@@ -444,9 +450,13 @@ class Scheduler {
   const QueueImplPtr &getDefaultHostQueue() const { return DefaultHostQueue; }
 
   static MemObjRecord *getMemObjRecord(const Requirement *const Req);
+  // Virtual for testing purposes only
```

Comment:
```suggestion
  // Virtual for testing purposes only
```
Is it still relevant?

Reply:
nope, fixed in https://github.com/intel/llvm/pull/6837/commits/71e9048af88ddcc853968cf9bbaa050f63ad323e